cmd/tailscale/cli/cli.go

```diff
@@ -51,6 +51,14 @@ func ActLikeCLI() bool {
 		return false
 	}
 
+	// Xcode adds the -NSDocumentRevisionsDebugMode flag on execution.
+	// If present, we are almost certainly being run as a GUI.
+	for _, arg := range os.Args {
+		if arg == "-NSDocumentRevisionsDebugMode" {
```

Comment:
Seems like this is only present if a certain option in Xcode is checked, but sure?

Reply:
Yeah, it's a fragile hack, but the whole thing is a pile of hacks, and the box is checked right now, so...mission accomplished.